### PR TITLE
Include endpoint urls in adapter and upstream sync

### PR DESCRIPTION
**Pull request by tphalp**

### CODE REVIEW

File: packages/sophtron-adapter/scripts/rename-esm.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+sleep 5
```
Comment (wesrisenmay-mx) [inline packages/sophtron-adapter/scripts/rename-esm.sh]: Why is this necessary?

File: packages/sophtron-adapter/src/index.ts [hunk 2]
```diff
@@ -8,12 +8,21 @@ import type { AdapterDependencies } from "./models";
 export const getSophtronAdapterMapObject = (
   dependencies: AdapterDependencies,
 ) => {
+  const deps = {
+    ...dependencies,
+    aggregatorCredentials: {
+      ...dependencies.aggregatorCredentials,
+      endpoint: "https://api.sophtron.com/api",
```
Comment (wesrisenmay-mx) [inline packages/sophtron-adapter/src/index.ts]: Feels gross passing around hardcoded endpoints in a credentials object

File: packages/sophtron-adapter/scripts/rename-esm.sh (hunk 1 above)
Comment (tphalp) [reply packages/sophtron-adapter/scripts/rename-esm.sh]: I found that sometimes this script errors when it runs, and the error is about files not existing. It seems the `tsc` build command that happens right before this script is run, needs a little more time for the files to show up in the file system before this can be run. I tried a couple other things, and this was the only one that was consistently successful.

Comment (tphalp) [reply packages/sophtron-adapter/scripts/rename-esm.sh]: I've removed the sleep and I'll keep an eye on this to see if we continue to run into this issue.

Comment (tphalp) [reply packages/sophtron-adapter/scripts/rename-esm.sh]: Still erroring. This is why I tried adding the sleep command: https://stackoverflow.com/a/78368373/97358

**Review by wesrisenmay-mx — APPROVED** [review]
